README.md

```diff
@@ -15,11 +15,56 @@ not yet verified to be rock-solid in a production environment.
 
 Tested with OTP releases ``R16B``, ``17`` and ``18``.
 
+## Installation
+
+### Rebar 3
+
+Craterl is built using rebar3. If you want to use it in your erlang 
+application, make sure you also use rebar3 and add it to your dependencies.
+
+If you want to load it from [hex.pm](https://hex.pm/packages/craterl) configure it like so:
+
+```erlang
+{deps,[
```

Comment:
@mfelsche Could be my Erlang ignorance… I add this to rebar.config, run rebar compile and get:

Dependency not available: craterl-0.2.3 (undefined)

If I try the git option I get:

ERROR: compile failed while processing /Users/chrisward/Workspace/erlangcrate: {'EXIT',{badarg,[{io,format,
                     [<0.24.0>,"Dependency not available: ~p-~s (~p)\n",
                      [craterl,
                       {git,"git://github.com/crate/craterl.git",
                            {tag,"0.2.3"}},
                       undefined]],
                     []},
                 {lists,foreach,2,[{file,"lists.erl"},{line,1337}]},
                 {rebar_deps,do_check_deps,1,
                             [{file,"src/rebar_deps.erl"},{line,164}]},
                 {rebar_deps,compile,2,
                             [{file,"src/rebar_deps.erl"},{line,134}]},
                 {rebar_core,run_modules,4,
                             [{file,"src/rebar_core.erl"},{line,493}]},
                 {rebar_core,execute,6,
                             [{file,"src/rebar_core.erl"},{line,418}]},
                 {rebar_core,maybe_execute,8,
                             [{file,"src/rebar_core.erl"},{line,302}]},
                 {rebar_core,process_dir1,7,
                             [{file,"src/rebar_core.erl"},{line,261}]}]}}